packages/flutter_driver/lib/src/extension/extension.dart

```diff
@@ -315,11 +322,14 @@ class FlutterDriverExtension with DeserializeFinderFactory, CreateFinderFactory,
   /// Creates an object to manage a Flutter Driver connection.
   FlutterDriverExtension(
     this._requestDataHandler,
-    this._silenceErrors, {
+    this._silenceErrors,
+    this._enableTextEntryEmulation, {
     List<FinderExtension> finders = const <FinderExtension>[],
     List<CommandExtension> commands = const <CommandExtension>[],
   }) : assert(finders != null) {
-    registerTextInput();
+    if (_enableTextEntryEmulation) {
+      registerTextInput();
+    }
```

Comment:
@nturgut Related to your reply here: https://github.com/flutter/flutter/pull/71656#issuecomment-743355197 -- it was `true` by default already before